query.go

```diff
@@ -25,8 +25,9 @@ const (
 // For the sake of backwards compatibility, when making changes to this type, ensure that changes are
 // only additive.
 type Query struct {
-	RawSQL string            `json:"rawSql"`
-	Format FormatQueryOption `json:"format"`
+	RawSQL string                 `json:"rawSql"`
+	Format FormatQueryOption      `json:"format"`
+	Args   map[string]interface{} `json:"args"`
```

Comment:
Could we use `json.RawMessage` here instead? `map[string]interface{}` is so hard to work with.